# Set your detection topic here
detection_topic: "/object_detection/detections"

# Set your camera topic here 
camera_topic: "/kinect2/qhd/image_color"

# Set your output topic here
output_topic: "/face_recognizer/faces"

# Set your output topic for RGB Image here
output_topic_rgb: "/face_recognizer/faces/image"
